content/docs/rendering-elements.md

```diff
@@ -1,75 +1,75 @@
 ---
 id: rendering-elements
-title: Rendering Elements
+title: Me-render Elemen
 permalink: docs/rendering-elements.html
 redirect_from:
   - "docs/displaying-data.html"
 prev: introducing-jsx.html
 next: components-and-props.html
 ---
 
-Elements are the smallest building blocks of React apps.
+Elemen adalah blok terkecil dalam aplikasi React.
 
-An element describes what you want to see on the screen:
+Sebuah elemen menggambarkan apa yang ingin anda lihat dalam layar:
 
 ```js
 const element = <h1>Hello, world</h1>;
 ```
 
-Unlike browser DOM elements, React elements are plain objects, and are cheap to create. React DOM takes care of updating the DOM to match the React elements.
+Tidak seperti elemen DOM, elemen pada React merupakan objek kosong, dan mudah dibuat. React DOM mengurus pembaharuan pada DOM untuk menyesuaikan dengan elemen pada React.
 
->**Note:**
+>**Catatan:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Anda mungkin akan sedikit bingung antara elemen dengan konsep yang lebih dikenali yaitu "komponen". Kami akan memperkenalkan komponen di [bagian berikutnya](/docs/components-and-props.html). Elemen adalah apa yang "membuat" komponen, dan kami menyarankan anda untuk membaca bagian ini sebelum melompat ke bagian berikutnya.
 
-## Rendering an Element into the DOM {#rendering-an-element-into-the-dom}
+## Me-*render* elemen ke dalam DOM {#rendering-an-element-into-the-dom}
 
-Let's say there is a `<div>` somewhere in your HTML file:
+Sebagai contoh terdapat sebuah `<div>` di suatu tempat di *file* HTML anda:
 
 ```html
 <div id="root"></div>
 ```
 
-We call this a "root" DOM node because everything inside it will be managed by React DOM.
+Kita menyebut ini sebagai "akar" *node DOM* karena semua yang berada di dalamnya nanti akan diatur oleh React DOM.
 
-Applications built with just React usually have a single root DOM node. If you are integrating React into an existing app, you may have as many isolated root DOM nodes as you like.
+Aplikasi yang dibuat dengan React biasanya memiliki satu akar *node DOM*. Jika anda mengintegrasikan React ke dalam aplikasi yang sudah ada, anda mungkin memiliki akar *node DOM* yang terisolasi sebanyak yang anda inginkan.
```

Comment:
```suggestion
Aplikasi yang dibuat dengan React biasanya memiliki satu *node DOM* akar. Jika anda mengintegrasikan React ke dalam aplikasi yang sudah ada, anda dapat memiliki *node DOM* akar terisolasi sebanyak yang anda inginkan.
```

WDYT? 😄

Reply:
@regalius I think both of them sounds fine to me, but your suggestion is better and it goes well with the sentence.